### PR TITLE
Add support for class= for framebox

### PR DESCRIPTION
**Pull request by surma**

### CODE REVIEW

File: devsiteHelper.py [hunk 1]
```diff
@@ -231,9 +231,10 @@ def renderDevSiteContent(content, lang='en'):
     fbContent = re.search(r'({%[ ]?framebox.+?%})(.*?){%[ ]?endframebox[ ]?%}(?ms)', framebox)
     fbOpenTag = fbContent.group(1)
     fbHeight = re.search(r'height="(.*?)"', fbContent.group(1))
+    fbClass = re.search(r'class="(.*?)"', fbContent.group(1))
     fbContent = fbContent.group(2)
     fbMemcacheKey = '/framebox/' + hashlib.md5(fbContent).hexdigest()
-    replaceWith = '<iframe class="framebox inherit-locale" '
+    replaceWith = '<iframe class="framebox inherit-locale ' + fbClass.group(1) + '" '
```
Comment (petele) [inline devsiteHelper.py]: If no `class` is found, `fbClass` ends up being `None`, and you can't call `fbClass.group(1)` on a `None`.

**Comment by surma** [comment]
@petele PTAL :)

File: devsiteHelper.py [hunk 2]
```diff
@@ -231,9 +231,13 @@ def renderDevSiteContent(content, lang='en'):
     fbContent = re.search(r'({%[ ]?framebox.+?%})(.*?){%[ ]?endframebox[ ]?%}(?ms)', framebox)
     fbOpenTag = fbContent.group(1)
     fbHeight = re.search(r'height="(.*?)"', fbContent.group(1))
+    fbClass = re.search(r'class="(.*?)"', fbContent.group(1))
     fbContent = fbContent.group(2)
-    fbMemcacheKey = '/framebox/' + hashlib.md5(fbContent.encode('utf-8')).hexdigest()
```
Comment (petele) [inline devsiteHelper.py]: Did you intend to remove the `.encode('utf-8')` bit here? You added it in #4499 just a few weeks ago.

Comment (surma) [inline devsiteHelper.py]: Uhm no, I definitely wasn’t intending to remove that. Must have resolved the conflict the wrong way around while rebasing -.- Thanks Pete!

**Comment by surma** [comment]
Fixed!

**Pull request merged**